index.html

```diff
@@ -15,208 +15,222 @@
+          </li>
+        </ul>
       </div>
-      <ul class="skills">
-        <li class="skill">
-          <img src="images/IconSkill.png" alt="" />
-          <h3>Languages</h3>
-          <ul class="list-tag">
-            <li class="tags">JavaScript</li>
-            <li class="tags">Ruby</li>
-            <li class="tags">Html</li>
-            <li class="tags">Css</li>
-          </ul>
-        </li>
-        <li class="skill">
-          <img src="images/IconFramework.png" alt="" />
-          <h3>Frameworks</h3>
-          <ul class="list-tag">
-            <li class="tags">Bootstrap</li>
-            <li class="tags">Ruby on rails</li>
-            <li class="tags">RSpec</li>
-            <li class="tags">Capybara</li>
-            <li class="tags">Selenium</li>
-          </ul>
-        </li>
-        <li class="skill">
-          <img src="images/Icon Langguage.png" alt="" />
-          <h3>Skills</h3>
-          <ul class="list-tag">
-            <li class="tags">Codekit</li>
-            <li class="tags">GitHub</li>
-            <li class="tags">Codepen</li>
-            <li class="tags">Gitlab</li>
-            <li class="tags">Terminal</li>
-          </ul>
-        </li>
-      </ul>
     </section>
 
     <footer>
-      <h2 class="section-title">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h2>
-      <form action="https://formspree.io/f/xoqbpave" method="POST">
-        <label for="name">
-          <input
-            id="name"
-            type="text"
-            placeholder="Name"
-            name="name"
-            maxlength="30"
-            required
-          />
-        </label>
-        <label for="email">
-          <input
-            id="email"
-            type="email"
-            placeholder="exmple@exemple.com"
-            name="email"
-            required
-          />
-        </label>
-        <label for="message">
-          <textarea
-            id="message"
-            maxlength="500"
-            placeholder="your message ..."
-            required
-          ></textarea>
-        </label>
-        <button type="submit" class="buttons btn-contact">Get in touch</button>
-      </form>
-
+      <div class="footer-wrapper">
+        <h2 class="section-title">
+          I'm always interested in hearing about new projects, so if you'd like to
+          chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/xoqbpave" method="POST">
+          <label for="name">
+            <input
+              id="name"
+              type="text"
+              placeholder="Name"
+              name="name"
+              maxlength="30"
+              required
+            />
+          </label>
+          <label for="email">
+            <input
+              id="email"
+              type="email"
+              placeholder="exmple@exemple.com"
+              name="email"
+              required
```

Comment:
- [ ] Good job, you have done excellent work, but some issues have not to address yet on your desktop version in the contact form area. please as much as you can follow the Figma template you choose. Fix :wrench: the background image in your design is not aligned properly. 
- [ ] Kindly fix the text alignment in the contact section of the paragraph. :smile_cat:  
- [ ] Kindly fix the text alignment in the contact section on the button. :+1: 

| Figma design | Your design |
|--------------------|-----------------|
|![Screenshot from 2022-11-02 18-56-37](https://user-images.githubusercontent.com/65129228/199548169-caaf8d89-09ef-4564-b85f-5e1abf114b66.png) |![Screenshot from 2022-11-02 19-55-27](https://user-images.githubusercontent.com/65129228/199553030-c35a7531-f375-4c1b-83e3-20445adb0f6b.png)|